components/LinkBase.tsx

```diff
@@ -28,6 +30,7 @@ export const LinkBase = ({
       className={classNames('btn m-2', className)}
```

Comment:
@deepakprabhakara I think better we remove the `m-2` from the Base component and add it to where it is needed.